src/libraries/System.Runtime.InteropServices/tests/ComInterfaceGenerator.Tests/ComInterfaceGenerator.Tests.csproj

```diff
@@ -7,19 +7,22 @@
     <EnableLibraryImportGenerator>true</EnableLibraryImportGenerator>
     <!-- These tests pull the attributes from Ancillary.Interop, so we don't need to include the attribute sources in this assembly. -->
     <IncludeLibraryImportGeneratorSources>false</IncludeLibraryImportGeneratorSources>
-    <!-- suppress warning when referencing architecture specific NativeExports.csproj -->
-    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
+    <ReferencesNativeExports>true</ReferencesNativeExports>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="$(CommonPath)DisableRuntimeMarshalling.cs" Link="Common\DisableRuntimeMarshalling.cs" />
   </ItemGroup>
-  
+
   <ItemGroup>
     <ProjectReference Include="..\..\gen\ComInterfaceGenerator\ComInterfaceGenerator.csproj" ReferenceOutputAssembly="false" OutputItemType="Analyzer" />
     <ProjectReference Include="..\Ancillary.Interop\Ancillary.Interop.csproj" />
     <ProjectReference Include="..\TestAssets\NativeExports\NativeExports.csproj" />
     <ProjectReference Include="..\TestAssets\SharedTypes\SharedTypes.csproj" />
   </ItemGroup>
-
+  <!-- Expose unmanaged entry points from NativeExports -->
+  <ItemGroup>
+    <UnmanagedEntryPointsAssembly Include="Microsoft.Interop.Tests.NativeExports" />
+    <DirectPInvoke Include="Microsoft.Interop.Tests.NativeExportsNE" />
```

Comment:
Aren't these automatically included by the Directory.Build.targets since ReferencesNativeExports=true?